contracts/TradetrustERC721.sol

```diff
@@ -32,14 +48,20 @@ contract TradeTrustERC721 is ERC721MintableFull, IERC721Receiver {
     emit TokenBurnt(_tokenId);
   }
 
+  function sendToNewTitleEscrow(address beneficiary, address holder, uint256 _tokenId) public onlyMinter {
+    // require(ownerOf(_tokenId) == address(this), "Cannot send token: Token not owned by token registry");
```

Comment:
Dont comment out code when you mean to delete them

Reply:
done. ok removed the commented code, will take note.